README.md

```diff
@@ -619,6 +619,8 @@ maintaining the Node.js project.
 **Pooja Durgad** &lt;Pooja.D.P@ibm.com&gt;
 * [RaisinTen](https://github.com/RaisinTen) -
 **Darshan Sen** &lt;raisinten@gmail.com&gt;
+* [himadriganguly](https://github.com/himadriganguly) -
+**Himadri Ganguly** &lt;himadri.tech@gmail.com&gt; (he/him)
```

Comment:
@himadriganguly - can you place these such that the sort order is maintained?

Reply:
Ok I will definitely do it.